src/params/devp_ltc.c

```diff
@@ -0,0 +1,116 @@
+/**
+ * LTC4015 through DeviceParamaters.
+ *
+ * Copyright Thinnect Inc. 2020
+ * @license MIT
+ */
+
+#include "devp_ltc.h"
+
+#include "stddef.h"
+
+#include "platform_io.h"
+#include "retargeti2cconfig.h"
+#include "platform_i2c.h"
+
+#include "ltc.h"
+
+// -----------------------------------------------------------------------------
+static int dp_ltc_battery_voltage_get (devp_t * param, void * value);
+
+static devp_t m_dp_ltc_battery_voltage =
+{
+    .name = "battery_voltage_(mV)",
```

Comment:
I think there is a kind of soft limit of 16 characters (+ \0) - reads work with longer names, but writes don't.